scripts/upload-assets.js

```diff
@@ -110,7 +110,9 @@ async function uploadAssets() {
     const type = f[1]; // get integration name
     const integration = f[2]; // get integration name
     const file = f[f.length - 1];
-    const clientName = file.substring(0, file.indexOf('.'));
+    const clientName = file.includes('analytics.js-') ? 
+      file.substring(0, file.lastIndexOf('.js')) :
+      file.substring(0, file.indexOf('.'));
```

Comment:
Is there any reason to not just do this -- basically, we grab anything before the last .js?

```
const clientName  = file.substring(0, file.lastIndexOf('.js'))
```

Reply:
That would affect the way our `.dynamic.js.gz` files are obfuscated and uploaded, which would require a change in how we fetch those files in ajs. It's doable, just would require a bit of a larger change to support and would still have some extra logic, just elsewhere.

Reply:
ah K... I didn't realize that was a case we had to handle. Be nice to add a comment in the code about what the expected filename formats are for future reference, since there are no tests.

Reply:
Done @silesky !